packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx

```diff
@@ -1539,58 +1539,6 @@ describe("MultiSelect", () => {
         });
     });
 
-    describe("error state styles", () => {
```

Comment:
Since we use css pseudo-classes now to handle hover/focus/active styling, these tests failed to verify style attributes because the state styling was applied using js previously. We have the Chromatic regression tests now to verify styling visually so I believe these are no longer necessary. Let me know though if you have any concerns with this!